docs/app/pages/components/sections/date-time-picker/integrated-date-picker-ng1/integrated-date-picker-ng1.component.html

```diff
@@ -84,6 +91,13 @@
                 and <code>end</code>.</td>
             <td>true</td>
         </tr>
+        <tr>
+            <td class="attribute">validator</td>
+            <td>function</td>
+            <td>variable</td>
+            <td>Set a funtion here to override the default validation checks with your own custom checks.</td>
```

Comment:
I assume this function is expected to return true or false depending on the validity, should maybe mention this.